libr/bin/format/pyc/marshal.c

```diff
@@ -1152,6 +1158,9 @@ static bool extract_sections_symbols(pyc_object *obj, RList *sections, RList *sy
 	if (!cobj->name->data) {
 		return false;
 	}
+	if (!cobj->consts) {
```

Comment:
Collapse into a single line

Reply:
do you mean `if (!cobj->consts) { return false; }`??

Reply:
why? what about the others? maybe i should use r2 macros for this. if_null_return_value.